python/simple/v1/app.py

```diff
@@ -0,0 +1,63 @@
+from ampli import *
+from plugins import MyEventIDPlugin
+
+# Initialize the Ampli instance with LoadOptions and LoadClientOptions
+ampli = Ampli()
```

Comment:
Can we export a default instance like we do in JS/TS? https://github.com/amplitude/ampli-examples/blob/main/node/typescript/src/ampli/index.ts#L878